specs/RuntimeSpec/Fees/Fees.md

```diff
@@ -0,0 +1,138 @@
+# Runtime Fees
+
+Runtime fees are measured in Gas. The price of gas is determined from the previous block header `block_header.inner_rest.gas_price`.
+Gas price will be discussed separately.
+
+When a transaction is converted into a Receipt. The signer account is charged for the full cost of the transaction.
```

Comment:
should we mention that what is charged at this moment is potentially more than the transaction fee due to pessimistic gas price and the overcharged part is later refunded?

Reply:
Yes, but it's part of gas price discussion